home/static/js/exam-card.js

```diff
@@ -0,0 +1,122 @@
+// Function to generate cards for each exam
```

Comment:
The better aproach is delare html and css in html file, then use django template to generate UI